content/rest/authentication/authenticating-to-the-rest-api.md

```diff
@@ -21,7 +21,7 @@ Many REST API endpoints require authentication or return additional information
 
 To authenticate your request, you will need to provide an authentication token with the required scopes or permissions. There a few different ways to get a token: You can create a {% data variables.product.pat_generic %}, generate a token with a {% data variables.product.prodname_github_app %}, or use the built-in `GITHUB_TOKEN` in a {% data variables.product.prodname_actions %} workflow.
 
-After creating a token, you can authenticate your request by sending the token in the `Authorization` header of your request. For example, in the folllowing request, replace `YOUR-TOKEN` with a reference to your token:
+After creating a token, you can authenticate your request by sending the token in the `Authorization` header of your request. For example, in the following request, replace `YOUR-TOKEN` with a reference to your token:
```

Comment:
#### Before
https://docs.github.com/en/rest/authentication/authenticating-to-the-rest-api?apiVersion=2022-11-28#about-authentication
<img width="699" alt="image" src="https://github.com/github/docs/assets/2119212/8206eba7-9d5e-4a5e-9679-dea90090f0e6">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/rest/authentication/authenticating-to-the-rest-api#about-authentication
<img width="690" alt="image" src="https://github.com/github/docs/assets/2119212/4f244717-09c2-4d90-b760-6643cd5eca01">